server/functions.lua

```diff
@@ -478,6 +478,57 @@ end
 
 exports('GetPlayersData', getPlayersData)
 
+---@param filters table<string, any>
+local function handleFilters(filters)
+    if not (filters) then return '', {} end
+    local holders = {}
+    local clauses = {}
+    if filters.citizenid then
+        clauses[#clauses + 1] = 'citizenid = ?'
+        holders[#holders + 1] = filters.citizenid
+    end
```

Comment:
I don't think we need this as you don't need to search if you already have a specific citizenid